docs/release-notes/relnotes-2012.md

```diff
@@ -41,3 +41,25 @@
 - (GDO-544) Resolved issue where PingDataGovernance PAP images' MAX_HEAP_SIZE variable had no effect.
 - (GDO-618) Resolved issue where base layer was missing JMX agent.
 - (GDO-640) Resolved issue where wait-for command didn't honor timeout when waiting for host:port.
+
+### Product Build Matrix
+
+The following table includes product versions and their accompanying Image build status for this release.
```

Comment:
This is a nit but did you mean to capitalize Image?  I probably wouldn't but it's not a huge deal